pytest_twisted.py

```diff
@@ -275,10 +288,27 @@ def init_qt5_reactor():
 def init_asyncio_reactor():
     from twisted.internet import asyncioreactor
 
-    _install_reactor(
-        reactor_installer=asyncioreactor.install,
-        reactor_type=asyncioreactor.AsyncioSelectorReactor,
-    )
+    try:
+        _install_reactor(
+            reactor_installer=asyncioreactor.install,
+            reactor_type=asyncioreactor.AsyncioSelectorReactor,
+        )
+    except NotImplementedError as e:
+        import asyncio
+
+        _, _, traceback_object = sys.exc_info()
+        stack_summary = traceback.extract_tb(traceback_object)
+        source_function_name = stack_summary[-1].name
+        event_loop = asyncio.get_event_loop()
+
+        if (
+                source_function_name == "add_reader"
+                and isinstance(event_loop, asyncio.ProactorEventLoop)
+        ):
+            six.raise_from(
+                value=AddReaderNotImplementedError.build(),
+                from_value=e,
+            )
```

Comment:
Needs some `else: raise` otherwise you suppress the exception

Reply:
Like I said, late and sleepy...  `:|`  Will do.